src/sentry/tasks/store.py

```diff
@@ -965,6 +960,7 @@ def save_event_attachments(
     )
 
 
+# TODO(swatinem): remove this (and related queue) once the backing worker deployment is gone
```

Comment:
Can you elaborate on this?

Reply:
I hope the description explains things, and that we are slowly removing this thing one part after the other.